examples/finnegans_wake_demo/finnegans-wake-demo-federated.py

```diff
@@ -61,55 +51,69 @@
 ##############################################
 # Ursula, the Untrusted Re-Encryption Proxy  #
 ##############################################
-ursula = Ursula.from_seed_and_stake_info(seed_uri=SEEDNODE_URI,
-                                         federated_only=True,
-                                         minimum_stake=0)
+ursula = Ursula.from_seed_and_stake_info(seed_uri=SEEDNODE_URI, federated_only=True)
 
 # Here are our Policy details.
-policy_end_datetime = maya.now() + datetime.timedelta(days=5)
+policy_end_datetime = maya.now() + datetime.timedelta(days=1)
 m, n = 2, 3
 label = b"secret/files/and/stuff"
 
+
+#####################
+# Bob the BUIDLer  ##
+#####################
+
+# First there was Bob.
+LOCAL_BOB = Bob(federated_only=True,
```

Comment:
Can't say I like all caps for regular variables. What's the rationale here? To distinguish the character objects?

Reply:
Actually, I'm a little confused. At some point there were two variables, `BOB` and `bob`, weren't there? My comment about `bob` and `remote_bob` applied then, but doesn't make sense now. If there's only one Bob, there's probably no need in emphasizing its locality. Sorry about that.

Reply:
>  At some point there were two variables, BOB and bob, weren't there?

Yep, `BOB` was renamed to `LOCAL_BOB`, and `bob` was renamed to `remote_bob`.  

> If there's only one Bob, there's probably no need in emphasizing its locality. Sorry about that.

It's no problem! :smile: Sometimes it takes a few iterations of revision and reading to get the best outcome.  There are lots of subtle communications and hints baked into this demo, it's turning out nicely.  I agree, let's try out just `alice` and `bob`, as normally names lowercase variables.  It's less dramatic/distracting.

Reply:
:heavy_check_mark: